src/helpers/fabric.rs

```diff
@@ -24,23 +24,15 @@ pub type MessageChunks = (ChannelId, Vec<MessageEnvelope>);
 #[async_trait]
 pub trait Network: Sync {
     /// Type of the channel that is used to send messages to other helpers
-    type Channel: CommunicationChannel;
+    type Sink: futures::Sink<MessageChunks, Error = Error> + Send + Unpin + 'static;
     type MessageStream: Stream<Item = MessageChunks> + Send + Unpin + 'static;
 
-    /// Returns a new connection to be open. `channel_id` indicates the parameters of this
-    /// connection (destination peer and step). Once the connection is returned it is immediately
-    /// ready for sending messages.
-    async fn get_connection(&self, channel_id: ChannelId) -> Self::Channel;
+    /// Returns a sink that accepts data to be sent to other helper parties.
+    fn sink(&self) -> Self::Sink;
```

Comment:
I think that we want this name to be a bit clearer in terms of its purpose.  send_stream(), perhaps?